### PR TITLE
added support to binary mathematical functions

### PR DESCRIPTION
**Pull request by NoamShaish**
added support to binary mathematical functions and added division, multiplication, addition and subtraction

### CODE REVIEW

**Comment by iconara** [comment]
Thank you for moving your PR here. Could you move the tests to their own file? I don't think it's appropriate to put your maths functions in a test called "…WithStringFunctions…"

File: src/main/java/io/burt/jmespath/contrib/function/MultipleFunction.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package io.burt.jmespath.contrib.function;
+
+public class MultipleFunction extends MathBiFunction {
```
Comment (iconara) [inline src/main/java/io/burt/jmespath/contrib/function/MultipleFunction.java:3]: This should be renamed "multiply".

**Review by iconara — COMMENTED** [review]
What should happen when someone divides by zero? Now I assume they'll get `ArithmeticException`. Is this the intention? Is it desirable? I don't know, and the JMESPath specification is very vague on this. Overall JMESPath seems to favour returning `null` from functions when something goes wrong.

**Comment by NoamShaish** [comment]
> What should happen when someone divides by zero? Now I assume they'll get `ArithmeticException`. Is this the intention? Is it desirable? I don't know, and the JMESPath specification is very vague on this. Overall JMESPath seems to favour returning `null` from functions when something goes wrong.

Since dividing double by Zero is doesnt throw ArithmeticException the actual result will be a primitive Json String with the value Infinity.

I adopted the code to return json null in case of infinity but that might be confusing in my opinion.
I dont like this obsession with nulls in JmesPath in general, but I guess its better to be consistent with the general behaviour.

**Review by iconara — APPROVED** [review]
Looks good. Thank you for adding a test for the divide by zero case.

I'll have to fix the CI and make sure things work, but I'll merge and make a new release as soon as I can.

**Comment by iconara** [comment]
@grddev can you override the CI check for this PR? The tests pass as far as I can tell, but I don't seem to have permission for changing settings or overriding checks anymore. Perhaps you can merge the PR too while you're at it.

**Comment by iconara** [comment]
@NoamShaish 0.4.0 has been released, see https://github.com/burtcorp/jmespath-java-contrib/releases/tag/jmespath-contrib-0.4.0

@grddev thank you for the help, now I need help with #7
